dependencies.yaml

```diff
@@ -217,33 +236,31 @@ dependencies:
           # This index is needed for cudf and rmm.
           - --extra-index-url=https://pypi.nvidia.com
           - --extra-index-url=https://pypi.anaconda.org/rapidsai-wheels-nightly/simple
+      - output_types: [requirements, pyproject]
+        packages:
+          - dask-cuda==24.6.*,>=0.0.0a0
+          - rapids-dask-dependency==24.6.*,>=0.0.0a0
```

Comment:
Do we actually need to split this off from `[conda, requirements, pyproject]` to `[requirements, pyproject]`? Or can we just leave the alpha spec in place for conda dependency specifiers, too?

Reply:
`conda search -c rapidsai-nightly 'cuml=24.6.*'` and `conda search -c rapidsai-nightly 'cuml=24.6.*,>=0.0.0a0'` turn up the same results, so it seems that we can leave the alpha specifiers in place if we want.